tests/integration/test_mailers.py

```diff
@@ -106,3 +106,8 @@ def test_reset_password_msg(self, send_mail_msg, new_user):
             Email.send_reset_password_msg(new_user)
 
         send_mail_msg.assert_called()
+
+    @patch.object(Mail, 'send')
+    def test_send_user_invite_msg(self, send_mail_msg, new_user):
+        Email.send_user_invite_msg(new_user)
```

Comment:
Can we get this line `with patch.object(Message, '__init__', return_value=None):` added above line 112 here. Without it this test will be really slow. 

```suggestion
        with patch.object(Message, '__init__', return_value=None):
            Email.send_user_invite_msg(new_user)
```

I think we would want to have some other tests here or at least a test in the integration test that checks that the right user is actually emailed, but I'm not to worried about at this time. I'm not sure if pytest offers some methods to easily do that either. We're going to need to switch to another emailer library so we will do it at that time.